### PR TITLE
add test example for network delay, bandwidth and partition in TiDB

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

File: network/delay.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+### Hypothesis
+
+When the network delay between TiDB and TiKV nodes increases by 50ms, the QPS/TPS of TiDB will drop significantly, but it can still provide services normally.
+
+### Preparation
+
+1. Install chaos-mesh.
+2. A TiDB cluster on k8s with at least 3 TiKV Pods, deploy with [TiDB operator](https://docs.pingcap.com/tidb-in-kubernetes/stable/tidb-operator-overview).
+3. Runing payload on TiDB cluster.
```
Comment (fewdan) [inline network/delay.md]: s/Runing/Running/g

Comment (WangXiangUSTC) [reply network/delay.md]: done

**Comment by Andrewmatilde** [comment]
I think we can provide more examples base on different effect of network chaos applied between two of tikvs, pds, tidbs pods.

**Comment by WangXiangUSTC** [comment]
> I think we can provide more examples base on different effect of network chaos applied between two of tikvs, pds, tidbs pods.

I add more examples, PTAL again @Andrewmatilde

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**